packages/ocs/block-pool/body.tsx

```diff
@@ -79,6 +86,63 @@ export const BlockPoolBody = (props: BlockPoolBodyPros) => {
     []
   );
 
+  const [data, loaded, loadError] = useK8sList(CephBlockPoolModel);
+
+  const existingNames = React.useMemo(() => {
+    return loaded && !loadError ? data?.map((data) => getName(data)) : [];
+  }, [data, loaded, loadError]);
```

Comment:
```suggestion
  const existingNames = React.useMemo(() => (
    loaded && !loadError ? data?.map((data) => getName(data)) : [];
  ), [data, loaded, loadError]);
```
 [a return is required when we keep more than one line in the function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)

Reply:
a return IS required within a {} block. using a {} block in an arrow function is a preference (when there's only one return line) and sometimes necessary when you need a breakpoint, which was my case. Regardless, I've removed the {} block and return.